packages/account/src/Sections/Security/AccountLimits/account-limits-info.tsx

```diff
@@ -1,9 +1,10 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { Icon, Text } from '@deriv/components';
 import { localize, Localize } from '@deriv/translations';
 
-const currency_name_map = {
+type TCurrency_name_map = Record<string, { display_code: string; name: string }>;
```

Comment:
@yauheni-deriv This type is redundant, Instead mark `currency_name_map` object `as const` 🙏🏻